app/controllers/songs_controller.rb

```diff
@@ -1,10 +1,25 @@
 class SongsController < ApplicationController
   def index
+    @songs = Song.all
   end
 
   def new
   end
 
   def edit
   end
+  
+  def search
+    @results = true
+    if params[:search].nil?
+      @songs = Song.all
```

Comment:
Might want to paginate so when we have 1000 songs, they don't all get loaded. Check out https://github.com/mislav/will_paginate